src/main/java/com/indeed/jiraactions/api/customfields/CustomFieldValue.java

```diff
@@ -69,7 +78,7 @@ public boolean isEmpty() {
                 // Intentional fall-through
             case SEPARATE:
             case USERNAME:
-                return ImmutableList.of(sanitize(getTransformedValue(value)), sanitize(getTransformedValue(childValue)));
+                return ImmutableList.of(sanitize(getTransformedValue(value)), sanitize(getTransformedValue(childValue)), sanitize(getTransformedValue(groupsValue)));
```

Comment:
Is this the right thing to do? Imagine we have a custom field that is a list of users. If you have 5 users with 10 groups, is the right thing 5 values, each containing a list of groups, or 5*10 values?